content/management/configuration-api/v3.2/index.mdx

```diff
@@ -2785,6 +2785,230 @@ curl -X POST \
 
 </Section>
 
+# Auto access rules
+
+Auto access rules allow to assign groups to customer session according to their tracking url and geolocation.  Those rules are
+checked on **Login** and **Update Customer Page** and resulting groups stored in customer session.  When that session starts chat with **Start Chat**
+or **Activate Chat** without `chat.access` stored session groups are used as new thread's access.
+
+## Methods
+
+<Section>
+
+<Text>
+
+### Add auto access rules
```

Comment:
Creates an `auto access` data structure, which is a set of conditions for the tracking URL and geolocation of a customer.